chrome/content/test/tests.js

```diff
@@ -10,14 +10,36 @@ const PrivlyXPCOM = Components.classes["@privly.test/jetpack;1"].
                       getService(Components.interfaces.nsISupports).
                       wrappedJSObject;
 const { ls } = PrivlyXPCOM.getLocalStorageShim();
-const { storage } = PrivlyXPCOM.getSimpleStorage();
```

Comment:
It looks like you are mixing a lot of boilerplate into your tests. You should separate the specs from the boilerplate since you can include the boilerplate before any specs are run.